static/app/views/settings/projectPerformance/projectPerformance.tsx

```diff
@@ -401,8 +414,8 @@ class ProjectPerformance extends AsyncView<Props, State> {
             </Access>
           </Form>
         </Feature>
-        <Feature features={['organizations:performance-issues-dev']}>
-          <Fragment>
+        <Fragment>
+          <Feature features={['organizations:performance-issues-dev']}>
```

Comment:
Can we unify these forms under the same feature flag? This feature flag also just gives the Perf Team/Sentry access to these form fields

Reply:
Hey @narsaynorath, thanks for the review. 

- The backend does not rely on the flag. 
- The old flag targets the sentry org, for the new one we want it to eventually be available for all orgs, it's just going to be visible to superusers.